corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,6 +1,6 @@
 from corehq.util.elastic import es_index
 
-DOMAIN_INDEX = es_index("hqdomains_20151204_1053")
```

Comment:
You also need to update the test cases which have hardcoded references to this index name - they're in this file: https://github.com/dimagi/commcare-hq/blob/master/testapps/test_pillowtop/tests/data/all-pillow-meta.json#L160